src/cairoWriter/writers/sourceUnitWriter.ts

```diff
@@ -57,13 +57,11 @@ export class SourceUnitWriter extends CairoASTNodeWriter {
         !(f instanceof CairoImportFunctionDefinition),
     );
 
-    const writtenImportFuncs = getGroupedImports(
-      importFunctions
-        .sort((funcA, funcB) =>
-          `${funcA.path}.${funcA.name}`.localeCompare(`${funcB.path}.${funcB.name}`),
-        )
-        .filter((func, index, importFuncs) => func.name !== importFuncs[index - 1]?.name),
-    ).reduce((writtenImports, importFunc) => `${writtenImports}\n${importFunc}`, '');
+    const writtenImportFuncs = importFunctions
+      .map((n) => writer.write(n))
+      .sort((importA, importB) => importA.localeCompare(importB))
+      .filter((func, index, importFuncs) => func !== importFuncs[index - 1])
+      .reduce((writtenImports, importFunc) => `${writtenImports}\n${importFunc}`, '');
```

Comment:
Btw, do I understand correctly the reduce could be replaced by `.join('\n')` @rodrigo-pino?

Reply:
Yep, totally. Again, I reused the old logic and didn't notice about this. Will change it now.

Reply:
Yea, I've spotted that's not your logic. Caught my eye though, so had to mention :)